src/containers/Admin/NameTab/CustomURL/CustomURL.tsx

```diff
@@ -122,21 +122,21 @@ const CustomURL = (): JSX.Element => {
     )
 
     return (
-        <div className="custom-url">
-            <Paragraph className="name-page__paragraph">
+        <div className={classes.CustomURL}>
+            <Paragraph>
                 Her kan du lage en selvbestemt lenke til tavla di. Dette gjør
                 det både lettere for deg óg andre å ha tilgang til tavla di.
             </Paragraph>
-            <Paragraph className="name-page__paragraph">
+            <Paragraph>
                 <EmphasizedText>
                     Obs, den gamle Tavla-lenken din vil slettes hvis du setter
                     en ny.
                 </EmphasizedText>
             </Paragraph>
             <Requirements />
-            <div className="input-area">
-                <Paragraph className="prepend">tavla.entur.no/t/</Paragraph>
-                <div className="input-field">
+            <div className={classes.InputArea}>
+                <Paragraph margin="none">tavla.entur.no/t/</Paragraph>
```

Comment:
Det er good å kjøre inline styling her?

Reply:
Tenker det går fint her siden det er en property av Paragraph istedet for faktisk inline styling (e.g. style="..."). Ser ut som den bare forandrer Paragraph til å ha class name eds-paragraph--margin-none eller noe lignende så vi får ikke specificity issues.